pyclaragenomics/samples/sample_cudapoa

```diff
@@ -38,11 +38,12 @@ def run_cudapoa(groups, msa, print_output):
 
     # Create batch
     max_sequences_per_poa = 100
-    gpu_mem_per_batch = 0.9 * free
+    max_seq_sz = 1024
     batch = cudapoa.CudaPoaBatch(max_sequences_per_poa,
-                                 gpu_mem_per_batch,
+                                 max_seq_sz,
+                                 0.9 * free,
                                  stream=None,
-                                 output_type=("msa" if msa else "consensus"))
+                                 output_mask=msa)
```

Comment:
I think this line didn't need to be changed? It's still called `output_type` in the API and takes in a string, right? And either "msa" or "consensus" based on sample options?